gapic/api/artman_common.yaml

```diff
@@ -0,0 +1,25 @@
+common:
```

Comment:
If we can depend on this file, why not get rid of the individual config files with the hardcoded values?